examples/moviegen/README.md

```diff
@@ -0,0 +1,348 @@
+
+## Architecture
+
+<details>
+<summary><b>Architecture details</b></summary>
+
+### Transformer Backbone
+
+The Movie Gen family of models contains the following variations: 1B, 5B, and 30B parameters.
+It uses the [LLaMa3](https://arxiv.org/abs/2407.21783) backbone architecture for the joint image-video generation model,
+enabling confident scaling of the model size while maintaining efficient training.
+
+There are three changes to the LLaMa3 Transformer block for the use case of video generation using Flow Matching:
+
+1. Add a cross-attention module between the self-attention module and the feed forward network (FFN)
+   to each Transformer block to incorporate text conditioning based on the text prompt embedding **P**.
+   Multiple different text encoders are leveraged due to their complementary strengths
+   (see [Text Encoders](#text-encoders)).
+2. Add adaptive layer norm blocks to incorporate the time-step t to the Transformer, as used in prior work
+   ([DiT](https://arxiv.org/abs/2212.09748)).
+3. Use full bidirectional attention instead of causal attention used in language modeling.
+
+### TAE
+
+[//]: # (TODO)
+
+### Text Encoders
+
+Movie Gen uses a combination of [UL2](https://arxiv.org/abs/2205.05131), [ByT5](https://arxiv.org/abs/2105.13626), and
+Long-prompt [MetaCLIP](https://arxiv.org/abs/2309.16671) as text encoders to provide both semantic-level and
+character-level text understanding for the backbone:
+
+- **UL2** is trained using massive text-only data and potentially provides strong text reasoning abilities in its
+  features.
+- **Long-prompt MetaCLIP** provides text representations that are aligned with visual representations that are
+  beneficial
+  for cross-modal generation.
+- **ByT5** encoder is only used to encode visual text, i.e., the part of the text prompt that explicitly asks for a
+  character string to be generated in the output image / video.
+
+</details>
+
+## Installation
+
+| MindSpore | Ascend Driver |  Firmware   | CANN toolkit/kernel |
+|:---------:|:-------------:|:-----------:|:-------------------:|
+|   2.3.1   |   24.1.RC2    | 7.3.0.1.231 |    8.0.RC2.beta1    |
+
+1. Install MindSpore according to the [official instructions](https://www.mindspore.cn/install).
+   For Ascend devices, please install
+   [CANN8.0.RC2.beta1](https://www.hiascend.com/developer/download/community/result?module=cann&cann=8.0.RC2.beta1)
+   and [MindSpore 2.3.1](https://www.mindspore.cn/install).
+2. Install requirements
+    ```shell
+    pip install -r requirements.txt
+    ```
+
+## Model Weights
+
+<details>
+<summary><b>TAE</b></summary>
```

Comment:
This is only needed for tae  training, not llamaDiT training. pls replace with the instruction to download the uploaded TAE checkpoint